### PR TITLE
Final update vignette malaysia

### PR DESCRIPTION
**Pull request by jacobvjk**
explanation of the `return_results` option

### CODE REVIEW

File: vignettes/articles/03-run-stress-test.Rmd [hunk 1]
```diff
@@ -65,6 +65,12 @@ This will calculate the stress test for loans and write the results along with a
 log file into the designated output directory, using the default values for all
 detail arguments.
 
+**NOTE:** The default mode will **not return** any objects within the R session.
+We find this is the easiest and most user friendly way to obtain results, but we
+acknowledge that more advanced R users may find this to depart from the
+conventions of other packages. Below we highlight, how to return the results as
```
Comment (MirijaHa) [inline vignettes/articles/03-run-stress-test.Rmd:71]: I would not say convention of other packages, this phrase does not make sense and I could banter on on why, instead I would suggest to say sth like>
We alternatively provide the option that the function returns a list holding the result objects. This allows the user to inspect and further analyse the data directly in R.

File: vignettes/articles/03-run-stress-test.Rmd [hunk 2]
```diff
@@ -106,6 +112,10 @@ company and technology should be excluded from the analysis. At this point, it
 is strictly recommended to set this to TRUE (default) because there is not yet a
 dedicated mechanism in place that handles companies which are phasing out
 certain technologies within the time frame of the production forecast.
+* `return_results` Logical parameter that indicates if the outputs of the
+analysis should be directly written to the indicated output path or simply
+returned as objects within the running session. The default is FALSE, which
```
Comment (MirijaHa) [inline vignettes/articles/03-run-stress-test.Rmd:117]: - list of results.
- I was wondering if to add a use example here, but then again who wants to use this option should be able to understand it without help.

File: vignettes/articles/03-run-stress-test.Rmd [hunk 3]
```diff
@@ -124,15 +134,27 @@ run_stress_test(
 )
 ```
 
+**NOTE:** One important input argument that affects the behavior of where the
+results are written, is the argument `return_results`. Per default it is set to
+`FALSE`, which means that results are written as csv files to a sub directory in
+the `output_path` and the user must open these files to view the results.
+If the user wants to view the results directly in RStudio, they can set the
```
Comment (MirijaHa) [inline vignettes/articles/03-run-stress-test.Rmd:141]: R, not RStudio

File: vignettes/articles/03-run-stress-test.Rmd [hunk 4]
```diff
@@ -124,15 +134,27 @@ run_stress_test(
...
+argument to `TRUE` which will return them as data frames in the R session. This
```
Comment (MirijaHa) [inline vignettes/articles/03-run-stress-test.Rmd:142]: list of dataframe

File: vignettes/articles/03-run-stress-test.Rmd [hunk 5]
```diff
@@ -124,15 +134,27 @@ run_stress_test(
...
+means they are not written to the designated `output_path` however, so the user
```
Comment (MirijaHa) [inline vignettes/articles/03-run-stress-test.Rmd:143]: if themselves then users

**Review by MirijaHa — COMMENTED** [review]
nits

File: vignettes/articles/03-run-stress-test.Rmd (hunk 1 above)
Comment (jacobvjk) [reply vignettes/articles/03-run-stress-test.Rmd:71]: Thinking about it, I believe we should consider a follow up (non blocking for the next week) to return the results as 5 separate objects. If we aim at user friendliness and expect that some users will not have used R much, I think a list type object still presents somewhat of a barrier.

Comment (MirijaHa) [reply vignettes/articles/03-run-stress-test.Rmd:71]: a list is the only way to cleanly return >1 objects 🤡  , unless we adjust the function in a way that it only returns 1 result type per run. 
I would say being able to deal with a list is a nicer selector for if one should use the option

File: vignettes/articles/03-run-stress-test.Rmd (hunk 5 above)
Comment (jacobvjk) [reply vignettes/articles/03-run-stress-test.Rmd:143]: I wanted to use a gender neutral they here. Does this imply I should use plural? I thought this worked different..

Comment (MirijaHa) [reply vignettes/articles/03-run-stress-test.Rmd:145]: ahh, i thought I deleted this comment, your formulation is correct afaik

File: vignettes/articles/03-run-stress-test.Rmd (hunk 1 above)
Comment (jacobvjk) [reply vignettes/articles/03-run-stress-test.Rmd]: fine by me, let's see what the users think

File: vignettes/articles/03-run-stress-test.Rmd (hunk 4 above)
Comment (jacobvjk) [reply vignettes/articles/03-run-stress-test.Rmd]: done

File: vignettes/articles/03-run-stress-test.Rmd (hunk 3 above)
Comment (jacobvjk) [reply vignettes/articles/03-run-stress-test.Rmd]: done

File: vignettes/articles/03-run-stress-test.Rmd (hunk 2 above)
Comment (jacobvjk) [reply vignettes/articles/03-run-stress-test.Rmd]: done (and no)

**Pull request merged**